### PR TITLE
Machine Learning Compute: Add optional deleteAll parameter

### PR DESCRIPTION
**Pull request by shutchings**
## Description
When deleting an operationalization cluster we leave behind the ACR, Application Insights, and Storage Accounts so the user has them for history. However some users do not want to delete these manually if they are not needed. The optional 'deleteAll' parameter adds the ability to delete all resources associated with the cluster as well.

[REST spec PR](https://github.com/Azure/azure-rest-api-specs/pull/2078)

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
1. Please get latest from upstream
2. Execute from root msbuild build.proj (this will update the build tools)
3. Then build your project from IDE or command line
msbuild build.proj /t:Build /p:Scope=SDKs\MachineLEarningCompute
4. Update PR

**Comment by shutchings** [comment]
Done. It looks like this created a new AzSdk.RP.props file. Are there any other expected changes?

**Comment by shutchings** [comment]
@shahabhijeet Could I get another look at this? My powershell cmdlet release deadline is coming up. Thanks!

File: src/SDKs/MachineLearningCompute/Management.MachineLearningCompute/Microsoft.Azure.Management.MachineLearningCompute.csproj [hunk 1]
```diff
@@ -7,9 +7,9 @@
         <PackageId>Microsoft.Azure.Management.MachineLearningCompute</PackageId>
         <Description>Provides developers with a library to create and manage machine learning compute resources.</Description>
         <AssemblyName>Microsoft.Azure.Management.MachineLearningCompute</AssemblyName>
-        <Version>0.2.0</Version>
-        <PackageTags>Microsoft Azure Machine Learning Compute management;</PackageTags>
-        <PackageReleaseNotes>Orchestrator properties and orchestrator service principal are optional since the resource provider will create one if not provided.</PackageReleaseNotes>
+        <Version>0.3.0</Version>
+        <PackageTags>Microsoft Azure Machine Learning Compute management;REST HTTP client;azureofficial;windowsazureofficial;netcore451511</PackageTags>
```
Comment (shahabhijeet) [inline src/SDKs/MachineLearningCompute/Management.MachineLearningCompute/Microsoft.Azure.Management.MachineLearningCompute.csproj]: @shutchings please remove all the extra tags that were removed earlier. Those tags does not make any sense.
All these tags are not required for searching your nuget package. Please remove them. We are cleaning these tags across all SDKs.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Comment by shutchings** [comment]
@shahabhijeet I removed the tags. Anything else?

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**